capella_console_client/validate.py

```diff
@@ -71,3 +73,21 @@ def _validate_and_filter_stac_ids(stac_ids: Optional[List[str]]) -> List[str]:
         logger.warning(f"filtered {','.join(duplicates)} (duplicate)")
 
     return valid_stac_ids
+
+
+def _snake_to_camel(snake):
+    REG = r"(.*?)_([a-zA-Z])"
+    return re.sub(REG, lambda x: x.group(1) + x.group(2).upper(), snake, 0)
+
+
+def _datetime_to_iso8601_str(
+    default_value: Optional[datetime], dt: Optional[Union[datetime, str]] = None
+) -> Tuple[datetime, str]:
+    if not dt:
+        dt = default_value
+    elif isinstance(dt, str):
+        try:
+            dt = parse(dt)
+        except ParserError:
+            raise ValueError(f"Could not parse {dt} string into a datetime")
+    return (dt, dt.isoformat(timespec="milliseconds") + "Z")  # type: ignore
```

Comment:
please only return `dt.isoformat(timespec="milliseconds") + "Z"` and adjust how `_datetime_to_iso8601_str` is used - actually I'll fix that myself